examle/cqlsh.py

```diff
@@ -0,0 +1,279 @@
+                    print(f"{Colors.RED}{e.args}{Colors.RESET}")
+                continue
+            statement = acsylla.create_statement(query, page_size=10)
+            result = None
+            try:
+                result = await self.session.execute(statement)
+            except (acsylla.errors.CassErrorServerSyntaxError, acsylla.errors.CassErrorServerInvalidQuery) as e:
+                print(f"{Colors.RED}{e.args}{Colors.RESET}")
+            count = 0
+            if result:
+                while result:
+                    count += len(result)
+                    self.print.table(result.columns_names(), [row.as_list() for row in result])
+                    if result.has_more_pages():
+                        try:
+                            input("\n---MORE---")
+                        except (EOFError, KeyboardInterrupt):
+                            print()
+                            break
+                        statement.set_page_state(result.page_state())
+                        result = await self.session.execute(statement)
+                        continue
+                    print(f"\n({count} rows)")
+                    break
+
+    async def stdin(self):
+        await self.init()
+        for query in sys.stdin:
+            statement = acsylla.create_statement(query, page_size=10)
+            result = await self.session.execute(statement)
+            if result:
+                writer = csv.writer(sys.stdout)
+                writer.writerow(result.columns_names())
+                while result:
+                    writer.writerows([row.as_list() for row in result])
+                    if result.has_more_pages():
+                        statement.set_page_state(result.page_state())
+                        result = await self.session.execute(statement)
+                    else:
+                        break
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Acsylla cqlsh example.")
+    parser.add_argument(
+        "contact_points", nargs="*", default=["127.0.0.1"], help="Set contact points, default: 127.0.0.1:9042"
+    )
+    parser.add_argument("-v", "--protocol-version", default=3, type=int)
+    parser.add_argument("-u", "--username", help="Authenticate as user.")
+    parser.add_argument("-p", "--password", help="Authenticate using password.")
+    parser.add_argument("-k", "--keyspace", help="Authenticate to the given keyspace.")
+    parser.add_argument(
+        "-V",
+        "--version",
+        action="version",
+        version=f"{Colors.BLUE}%(prog)s: {__version__}{Colors.RESET}{Colors.GREEN}acsylla: "
+        f"{acsylla.version.__version__}",
+    )
+    parser.add_argument("--ssl", help="Use SSL.", default=False, type=bool, dest="ssl_enabled")
+
+    cqlsh = AcsyllaCQLSH(parser.parse_args())
+    if sys.stdin.isatty():
+        asyncio.run(cqlsh.tty())
+    else:
+        asyncio.run(cqlsh.stdin())
```

Comment:
loved this example, it's by far much more simple then cqlsh current implementation